docs/newcontent/.pages:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+ReferrerUrl=C:\Users\KevinRoche\Downloads\kubestellar-ksdocs.zip

```

Comment:
This looks like something that does not belong in git. Same for all the other :Zone.Identifier files.

Reply:
Clearly I screwed up something in my commit.